server/game/cards/02_SHD/units/FreetownBackup.ts

```diff
@@ -0,0 +1,30 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType, WildcardZoneName } from '../../../core/Constants';
+
+export default class FreetownBackup extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4534554684',
+            internalName: 'freetown-backup'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'Give another friendly unit +2/+2',
+            optional: true,
```

Comment:
This ability isn't optional, it doesn't say "you may"

Reply:
good catch, I noticed it everywhere except there!